tools/scripts/utils.mjs

```diff
@@ -665,3 +743,84 @@ export async function checkEmptyDirectories(dirPath) {
 
   return emptyDirs;
 }
+
+export function getCosmosAddress(pubkeyBuffer) {
+  return toBech32(
+    'cosmos',
+    rawSecp256k1PubkeyToRawAddress(Secp256k1.compressPubkey(pubkeyBuffer))
+  );
+}
+
+export function getPubKeyBuffer(pubKey) {
+  if (pubKey.startsWith('0x')) {
+    pubKey = pubKey.slice(2);
+  }
+  return Buffer.from(pubKey, 'hex');
+}
+
+export async function signAuthMessage(
+  privateKey,
+  statement = 'TESTING TESTING 123'
+) {
+  const wallet = new ethers.Wallet(privateKey);
+
+  // expirtaion time in ISO 8601 format
+  const expirationTime = new Date(
+    Date.now() + 1000 * 60 * 60 * 24 * 7
+  ).toISOString();
+
+  const domain = 'localhost';
+  const origin = 'https://localhost/login';
```

Comment:
Should we allow defining the domain and origin?

Reply:
https://github.com/LIT-Protocol/js-sdk/blob/952f7abecdafc451599c6734e6a6a3c2bb0b975b/tools/scripts/utils.mjs#L783-L784